src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/ProbabilityTable/Builder/test/Builder.test.cpp

```diff
@@ -0,0 +1,156 @@
+            std::vector< double > aceified{};
+            aceified.push_back( 3 );
+            aceified.push_back( 2 );
+            aceified.push_back( INT );
+            aceified.push_back( inelasticCompetition );
+            aceified.push_back( otherAbsorption );
+            aceified.push_back( static_cast< int > ( factors ) );
+            aceified |= ranges::action::push_back( energies );
+            aceified |= ranges::action::push_back( CDFs );
+            aceified |= ranges::action::push_back( totalXS );
+            aceified |= ranges::action::push_back( elasticXS );
+            aceified |= ranges::action::push_back( fissionXS );
+            aceified |= ranges::action::push_back( captureXS );
+            aceified |= ranges::action::push_back( heating );
+            aceified |= ranges::action::push_back( CDFs );
+            aceified |= ranges::action::push_back( totalXS );
+            aceified |= ranges::action::push_back( elasticXS );
+            aceified |= ranges::action::push_back( fissionXS );
+            aceified |= ranges::action::push_back( captureXS );
+            aceified |= ranges::action::push_back( heating );
+
+            Table::Data data{};
+            pTable.ACEify( data );
+
+            CHECK( ranges::equal( aceified, data.XSS() ) );
+          }
+        }
+      }
+    }
+    WHEN( "adding probability table without heating or fission values" ){
+        tb.interpolationParameter( INT )
+          .inelasticCompetition( inelasticCompetition )
+          .otherAbsorption( otherAbsorption )
+          .factors( factors )
+          .incidentEnergies( njoy::utility::copy( energies ) )
+          .table()
+            .CDFs( njoy::utility::copy( CDFs ) )
+            .totalCrossSections( njoy::utility::copy( totalXS ) )
+            .elasticCrossSections( njoy::utility::copy( elasticXS ) )
+            .captureCrossSections( njoy::utility::copy( captureXS ) )
+          .add()
+          .table()
+            .CDFs( njoy::utility::copy( CDFs ) )
+            .totalCrossSections( njoy::utility::copy( totalXS ) )
+            .elasticCrossSections( njoy::utility::copy( elasticXS ) )
+            .captureCrossSections( njoy::utility::copy( captureXS ) )
+          .add()
+          .table()
+            .CDFs( njoy::utility::copy( CDFs ) )
+            .totalCrossSections( njoy::utility::copy( totalXS ) )
+            .elasticCrossSections( njoy::utility::copy( elasticXS ) )
+            .captureCrossSections( njoy::utility::copy( captureXS ) )
+          .add();
+
+        auto pTable = tb.construct();
+
+        THEN( "the values can be verified" ){
+          CHECK( INT == pTable.interpolationParameter );
+          CHECK( inelasticCompetition == pTable.inelasticCompetition );
+          CHECK( otherAbsorption == pTable.otherAbsorption );
+          CHECK( factors == pTable.factors );
+          CHECK( ranges::equal( energies, pTable.incidentEnergies ) );
+        }
```

Comment:
Test ACEify in this case as well.

Reply:
I've removed this entire `WHEN` into the above and just added a table without fission and heating.